apps/sms/views/conversation/js/compose.js

```diff
@@ -733,7 +733,14 @@ var Compose = (function() {
           dom.message.removeChild(currentAttachmentDOM);
           disposeAttachmentNode(currentAttachmentDOM);
           onContentChanged(newAttachment);
-        }, this._onAttachmentRequestError);
+        }, (err) => {
+          var errId = err instanceof DOMError ? err.name : err.message;
```

Comment:
Well, honestly I don't really like this solution due to following reasons:

1. When user taps on Cancel it's quite noticeable that menu was hidden and re-appeared that looks a bit clumsy;
2. It doesn't handle the case when activity was cancelled from the activity app. In this case ```pick cancelled``` error is returned (it would be great to confirm this case with UX as well, but not sure whom we can ask about it :( Is Morpheus our new UX?);
3. And it just looks too hackish to manually call ```onAttachmentClick``` in error handler.

Maybe we can modify OptionMenu instead? I was thinking about ```hideOnAction``` option (let's say for the entire OptionMenu at first, but can probably be on item level as well), which default value is ```true``` (to not introduce breaking change). In our case we just pass this pass ```options.hideOnAction: false``` and hide menu manually in all these cases. It would be easier if ```hideOnAction``` was on item option level, but I'm not sure if it's too much.

What do you think? Maybe better ideas exist as well.